plugins/modules/cloud/misc/terraform.py

```diff
@@ -93,6 +93,12 @@
     type: list
     elements: path
     version_added: '0.2.0'
+  check_destroy:
+    description:
+      - Check and apply only when there are no deletes to resources.
+    type: bool
+    default: false
+    version_added: '0.2.0'
```

Comment:
This needs to be changed to
```suggestion
    version_added: '1.0.0'
```

Reply:
sure @felixfontein , will do that